src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -100,10 +98,12 @@ void CalculatorProgrammerRadixOperators::IsErrorVisualState::set(bool value)
     }
 }
 
-void CalculatorProgrammerRadixOperators::OnViewModelPropertyChanged(Object^ sender, PropertyChangedEventArgs^ e)
+String^ CalculatorProgrammerRadixOperators::ParenthesisCountToString(unsigned int count) {
+    return count == 0 ? ref new String(L"") : ref new String(to_wstring(count).data());
+}
+
+
+void CalculatorProgrammerRadixOperators::CalculatorProgrammerRadixOperators::OpenParenthesisButton_GotFocus(Platform::Object^ sender, Windows::UI::Xaml::RoutedEventArgs^ e)
```

Comment:
Here and in the rest of the cpp files, you can just use Object^ instead of the full Platform::Object^

Reply:
cpp files are full of `Windows::UI::Xaml::RoutedEventArgs` and `Platform::Object` (because of the XAML editor), I will do a full cleaning and remove all of them